src/shogun/multiclass/CoverTreeKNNSolver.cpp

```diff
@@ -58,7 +57,7 @@ CMulticlassLabels* CCoverTreeKNNSolver::classify_objects(CDistance* distance, co
 	v_array< v_array< CJLCoverTreePoint > > res;
 	k_nearest_neighbor(top, top_query, res, m_k);
 
-#ifdef DEBUG_KNN
+#ifdef SG_DEBUG
```

Comment:
noup this is not correct... you wanna check the current log_level on runtime, not on compile time...